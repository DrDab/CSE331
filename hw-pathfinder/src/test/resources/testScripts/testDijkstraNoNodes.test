# Test if Dijkstra produces correct output on graph with no nodes.

# Create graph g
CreateGraph g

# Test pathfinding
FindPath g Seattle LosSantos
