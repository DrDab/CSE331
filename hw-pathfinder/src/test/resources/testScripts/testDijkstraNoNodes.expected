# Test if Dijkstra produces correct output on graph with no nodes.

# Create graph g
created graph g

# Test pathfinding
unknown: Seattle
unknown: LosSantos
